app/assets/javascripts/views/feedbacks.js

```diff
@@ -55,6 +35,42 @@ var previewAndSubmit = function() {
   });
 };
 
+var openNextAccordionPanel = function() {
+  var $accordion = $("#accordion").accordion();
+  var current = $accordion.accordion("option", "active"),
+      maximum = $accordion.find("h3").length,
+      next = current+1 === maximum ? 0 : current+1;
+  $accordion.accordion("option", "active", next);
+};
+
+var validateRequiredFields = function() {
+  var projectField = $('#feedback_project_worked_on').val();
+  var roleField = $('#feedback_role_description').val();
+  if(!projectField || !roleField) {
+    $('#accordion').accordion('activate', 0);
+    _addRequiredFieldClass(['#feedback_project_worked_on', '#feedback_role_description']);
+  }
+};
+
+var _addRequiredFieldClass = function(fields) {
+  for(var i in fields) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.